### ISSUE TITLE
Telegram voice calls ports for firewall

### ISSUE THREAD

**Issue by UnoSD** [TelegramMessenger/Telegram-iOS#263]
Hi, I am going through the documentation, but I don't seem to be able to find what ports are required for voice calls.

When I set up my firewall I would like to allow this traffic, can you please share (or update the docs) with the ports used?

Thank you.

**Comment by ethylmorphine** [comment]
I believe Telegram uses random ports from a range to dodge ISP blocks. You don't need to forward them though because I'm pretty sure they handled via upnp.

**Comment by UnoSD** [comment]
> I believe Telegram uses random ports from a range to dodge ISP blocks. You don't need to forward them though because I'm pretty sure they handled via upnp.

I don't think that would work for me, I'm using the guest network of the router and it won't support that.

**Comment by arthurrogado** [comment]
I need this too! I'm trying to allow this traffic on firewall. You got it something?

**Comment by acidwars** [comment]
I just ran into the same issue, I could observe the following ports being blocked on my firewall:

- 1400/udp
- 4000-5000/udp

after enabling these, calls worked.
